iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -236,7 +238,7 @@ typedef struct MQTTTRANSPORT_HANDLE_DATA_TAG
     int http_proxy_port;
     char* http_proxy_username;
     char* http_proxy_password;
-    bool isProductInfoSet;
+    bool isOptionalParameterSet;
```

Comment:
>isOptionalParameterSet [](start = 9, length = 22)

I think "optionalParameter" should be more specific.  Something like isOptionalConnect or some such?

Reply:
OptionalConnectParameter?

---
In reply to: [385380333](https://github.com/Azure/azure-iot-sdk-c/pull/1439#discussion_r385380333) [](ancestors = 385380333)

Reply:
Sure, but should proabbly actually be like isOptionalConnectParameterSet?

Reply:
yes

---
In reply to: [385381471](https://github.com/Azure/azure-iot-sdk-c/pull/1439#discussion_r385381471) [](ancestors = 385381471)